taskcluster/client.py

```diff
@@ -536,6 +536,11 @@ def _makeHttpRequest(self, method, route, payload):
 
             # Try to load JSON
             try:
+                if hasattr(response, 'headers'):
+                    if 'application/json' not in response.headers.get('Content-Type', '').split(';'):
```

Comment:
This is definitely correct to split on the `;` character and it correctly finds `application/json`, but the [spec](https://tools.ietf.org/html/rfc7231#section-3.1.1.1) only describes the mimetype as being before the first `;`.  How about making the comparison to the first index only?

Reply:
Hi John, is the way to run it still the same?

```
pip install tox
git clone https://github.com/taskcluster/taskcluster-client.py
cd taskcluster-client.py
tox -e py27  # this is just to get started, eventually you should also try just "tox"
```

I got a py27 command failed thus I cant test if my change mess the test or not

Reply:
yep!  that's what's needed to run it on my Fedora 28 machine.  Which OS and which pythons do you have installed?  Also, what's the error?

Reply:
yep!  that's what's needed to run it on my Fedora 28 machine.  Which OS and which pythons do you have installed?  Also, what's the error?

Reply:
I got this ERROR: InvocationError for command '/Users/angelinaf/Documents/taskcluster-client.py/.tox/py27/bin/python devDep.py' (exited with code 1) I use macOS. I remember previously when I open this PR i uses ubuntu and it works just fine. I will try to find why and get back to you

Reply:
you could also try install python through homebrew, the system python on os x is not so great.